### PR TITLE
ha: add RelayConfig related function

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of support `start-relay` of one source to many workers

### What is changed and how it works?
add etcd key prefix, k/v: worker-name -> source-id

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change
 - Has persistent data change

Side effects

 - Increased code complexity

Related changes

 - Need to cherry-pick to the release branch
 - Need to be included in the release note

### CODE REVIEW

File: pkg/ha/relay.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	ops := make([]clientv3.Op, 0, len(workers))
+	for _, worker := range workers {
+		ops = append(ops, deleteRelayConfigOp(worker))
+	}
+	_, rev, err := etcdutil.DoOpsInOneTxnWithRetry(cli, ops...)
+	return rev, err
+}
+
+// GetRelayConfig returns the source config which the given worker need to pull relay log from etcd, with revision
+func GetRelayConfig(cli *clientv3.Client, worker string) (*config.SourceConfig, int64, error) {
```
Comment (lance6716) [inline pkg/ha/relay.go:52]: almost same with https://github.com/pingcap/dm/blob/5141e05de7eae2b049c3bcba8f37c8874e4beb51/pkg/ha/bound.go#L167

**Review by lichunzhu — COMMENTED** [review]
Will one worker be bound with several relay sources?
Rest LGTM

**Comment by lance6716** [comment]
> Will one worker be bound with several relay sources?
currently not. User may need to deploy more worker if he want to increase relay replica number

**Review by GMHDBJD — APPROVED** [review]
lgtm

**Pull request merged**